src/o365/graph/commands/planner/task-list.ts

```diff
@@ -0,0 +1,138 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import Utils from '../../../../Utils';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import { Task } from './task';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  userid?: string;
```

Comment:
To be consistent, the option name should be `userId` (capital I)

Reply:
I fixed this, please check.